core/src/test/scala/org/apache/spark/sql/delta/optimize/OptimizeMetricsSuite.scala

```diff
@@ -64,6 +67,8 @@ trait OptimizeMetricsSuiteBase extends QueryTest
       assert(metrics.totalFilesSkipped == 0)
       assert(metrics.numTableColumns == 2)
       assert(metrics.numTableColumnsWithStats == 2)
+
+      assertOperationName(deltaLog)
```

Comment:
We don't need to verify the operation in every test. I think verifying the operation name as part of the tests that verify the output on history should be sufficient.

`optimize operation metrics in Delta table history`
`optimize ZOrderBy operation metrics in Delta table history`
`deletion vector metrics - optimize/zorder`